packages/broadcast/component.js

```diff
@@ -48,30 +53,37 @@ export class FabricBroadcast extends LitElement {
   }
 
   async _del(id) {
-    const el = this.renderRoot.querySelector(`#${id}`);
+    const el = this.renderRoot.querySelector(`#broadcast-${id}`);
     await el.collapse();
+    this.hiddenMessageIds = [...this.hiddenMessageIds, id];
   }
 
   render() {
+    const messages =
+      this.hiddenMessageIds.length > 0
+        ? this._messages.filter((item) => !this.hiddenMessageIds.includes(item.id))
+        : this._messages;
```

Comment:
Given that _messages and hiddenMessageIds area likely never more than a couple items at any given time, you could safely just always run the filter to slim down the code a bit

```js
const messages = this._messages.filter((item) => !this.hiddenMessageIds.includes(item.id))
```

Reply:
[9da5ddb](https://github.com/fabric-ds/elements/pull/61/commits/9da5ddbf339fa80aa2226fa0ec4834263b006ee3)